aws/resource_aws_eks_cluster.go

```diff
@@ -114,6 +116,14 @@ func resourceAwsEksCluster() *schema.Resource {
 					},
 				},
 			},
+			"logging": {
```

Comment:
We should consider renaming this attribute `cluster_log_types` or even `enabled_cluster_log_types` to show the designation that this handling is specifically for the `Types` within the `ClusterLogging` of the API. It also needs documentation added in `website/docs/r/eks_cluster.html.markdown` 👍